front_end/core/host/RNPerfMetrics.ts

```diff
@@ -5,26 +5,20 @@
 
 export type RNReliabilityEventListener = (event: ReactNativeChromeDevToolsEvent) => void;
 
-type UnsunscribeFn = () => void;
-export type RNPerfMetrics = {
-  addEventListener: (listener: RNReliabilityEventListener) => UnsunscribeFn,
-  removeAllEventListeners: () => void,
-  sendEvent: (event: ReactNativeChromeDevToolsEvent) => void,
-};
-
 let instance: RNPerfMetrics|null = null;
 
 export function getInstance(): RNPerfMetrics {
   if (instance === null) {
-    instance = new RNPerfMetricsImpl();
+    instance = new RNPerfMetrics();
   }
   return instance;
 }
 
-class RNPerfMetricsImpl implements RNPerfMetrics {
+type UnsunscribeFn = () => void;
```

Comment:
misspelling. Also can this be inlined, since it's not exported?

Reply:
Oof, fixed typo later in the stack but missed this one.

This is still exported transitively via `RNPerfMetrics`. I couldn't find a way to type-name the returned function inline at `addListener(…)`.

Can put it in a comment instead if you mean the while type alias should be removed. I think it's good to document the intent somewhere. `useEffect` names their clean up function as `Destructor`, for example.